encoding/codecs/x264.md

```diff
@@ -0,0 +1,292 @@
+# Encoding with x264
+
+H.264 has been the de facto standard video format
+across the internet for the past decade.
+It is widely supported for playback in all modern browsers
+and many hardware devices such as gaming consoles and phones.
+It provides better video quality at smaller file sizes
+compared to its predecessors.
+
+x264 is a mature, free, open-source encoder
+for the H.264 video format.
+This guide will focus on x264.
+
+## Prerequisites
+
+To get started, you'll need two things:
+
+- A video to encode--for the examples,
+  we will pipe in a video from VapourSynth,
+  which you should be able to do
+  if you've been following the previous sections of this guide
+- The x264 encoder
+
+Here's how we get a copy of the x264 encoder:
+
+### Windows
+
+Official Windows builds are available
+[here](https://download.videolan.org/pub/x264/binaries/win64/).
+
+### Linux/macOS
+
+Generally, x264 will be available
+through your distribution's package manager.
+Here are a few examples:
+
+- **Ubuntu/Debian**: `sudo apt install x264`
+- **Arch Linux**: `sudo pacman -S x264`
+- **macOS**: `brew install x264`
+
+## Getting Started
+
+x264 is very configurable,
+and the options may seem overwhelming.
+But you can get started encoding
+by using the presets x264 provides
+and understanding a few basic concepts.
+We'll walk through those concepts
+with the following examples.
+
+## Example 1: General-Purpose Encoding
```

Comment:
These examples should be on a third-level heading I think, as `Getting Started` is 2 and the commands are 4.

### Currently
![image](https://user-images.githubusercontent.com/10972040/66487229-f4c4ee80-ea79-11e9-84cb-a7f66ee3fbdf.png)


### Changed: 
![image](https://user-images.githubusercontent.com/10972040/66487146-db23a700-ea79-11e9-8b96-52f7b4777f5a.png)

### what it would look like if the commands were changed to level 5
![image](https://user-images.githubusercontent.com/10972040/66487448-5422fe80-ea7a-11e9-937a-3d2354f9ca10.png)